index.html

```diff
@@ -1671,12 +1677,15 @@ <h3>
         </p>
         <p>
           If the user agent honors the value of the <code>theme_color</code>
-          member as the <a>default theme color</a>, then that color serves as
-          the <a>theme color</a> for all browsing contexts to which the
-          manifest is <a>applied</a>. However, a document may override the
-          <a>default theme color</a> through the inclusion of a valid [[HTML]]
+          member as the [=default theme color=], then that color serves as the
+          [=theme color=] for all browsing contexts to which the manifest is
+          [=applied=]. However, the user agent MAY override the [=default theme
+          color=] if a document whose URL is [=within scope=] of the
+          [=application context=]'s [=manifest=] includes a valid [[HTML]]
           <code>meta</code> element whose <code>name</code> attribute is
-          "theme-color".
+          "theme-color". The user agent SHOULD NOT override the [=default theme
```

Comment:
```suggestion
          "theme-color". However, the user agent SHOULD NOT override the [=default theme
          color=] via an [^meta^] element whose [^meta/name^] attribute is "theme-color" for
          documents that are not [=within scope=], since the application has no control over
          these documents.
```